tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,199 @@
+"""
+.. _tut-fnirs-processing:
+
+Preprocessing fNIRS data
+========================
+
+This tutorial covers how to convert fNIRS data from raw measurements to
+relative oxyhaemoglobin (HbO) and deoxyhaemoglobin (HbR) concentration.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+Here we will work with the :ref:`fNIRS motor data <fnirs-motor-dataset>`.
+"""
+# sphinx_gallery_thumbnail_number = 3
+
+import os
+import mne
+from itertools import compress
+import matplotlib.pyplot as plt
+from mne.io.pick import _picks_to_idx
+
+
+fnirs_data_folder = mne.datasets.fnirs_motor.data_path()
+fnirs_raw_dir = os.path.join(fnirs_data_folder, 'Participant-1')
+raw_intensity = mne.io.read_raw_nirx(fnirs_raw_dir, verbose=True).load_data()
+
+
+###############################################################################
+# Selecting channels appropriate for detecting neural responses
+# -------------------------------------------------------------
+#
+# First we select optodes that are sufficiently separated to detect neural
+# responses (approximately 3cm) for further processing.
+
+long_channels = ~mne.preprocessing._beer_lambert_law._short_channels(
```

Comment:
> this is using a private function in a private submodule. Is there a public function we can use instead? If not, either make this public, or make a public wrapper for it.

It will be a commonly used function, so I agree lets make it public. I will do this in another PR and will also add a `long_channels` function as that will be useful too. 

For reference, the distance between optodes is related to how deep in the brain you are probing, so it is useful to be able to segregate your channels in to different distances that hopefully correspond to different depths.